src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -74,7 +74,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     if (!userStore.isUserRegistered(username)) {
-      User user = new User(UUID.randomUUID(), username, Instant.now());
+	  // TODO: add password
+      User user = new User(UUID.randomUUID(), username, null, Instant.now());
```

Comment:
is the value null because they create a password?

Reply:
/why did you add the "null" value

Reply:
The null value is a placeholder because the constructor of User needs to take in 3 arguments now. It will be resolved later after we can add and retrieve password from the database.